src/robust_provider/provider.rs

```diff
@@ -111,20 +111,17 @@ impl<N: Network> RobustProvider<N> {
     /// # Errors
     ///
     /// See [retry errors](#retry-errors).
+    #[instrument(level = "debug", skip(self))]
```

Comment:
```suggestion
    #[instrument(level = "tracing", skip(self))]
```
Let's support the lowest level for all `instrument` attributes, we don't know how devs will invoke this